static/app/views/settings/settingsIndex.tsx

```diff
@@ -80,7 +80,9 @@ class SettingsIndex extends React.Component<Props> {
     const supportText = isOnPremise ? t('Community Forums') : t('Contact Support');
 
     return (
-      <DocumentTitle title={organization ? `${organization.slug} Settings` : 'Settings'}>
+      <SentryDocumentTitle
+        title={organization ? `${organization.slug} Settings` : 'Settings'}
```

Comment:
```suggestion
        title={t('Settings')}
        orgSlug={organization?.slug}
```

Reply:
This one might make sense to keep as `my-org Settings - Sentry` maybe? vs `Settings - my-org - Sentry`

Reply:
maybe not, I might agree here

Reply:
Will get this in a 2nd pr